opening_hours.js

```diff
@@ -6728,7 +6728,7 @@
         function getWeekNumber(d) {
             // Copy date so don't modify original
             d = new Date(+d);
-            d.setHours(0,0,0);
+            d.setHours(0,0,0,0);
```

Comment:
just tried that.
indeed, if you DO use timekeep - the bug doesn't appear.
the moment i disable timekeep - both tests fail :|

with timekeep
```
"Week range. Working with Objects not Strings. from = new Date(Date())" for "week 01-53/2 Mo-Su 07:30-08:00": PASSED
"Week range. Working with Objects not Strings. from = new Date()" for "week 01-53/2 Mo-Su 07:30-08:00": PASSED
2/2 tests passed. 0 did not pass.
```

without timekeep
```
"Week range. Working with Objects not Strings. from = new Date(Date())" for "week 01-53/2 Mo-Su 07:30-08:00": FAILED, bad duration(s): 0,0, expected 1800000,0, bad intervals: 
(none)
expected:
[ '2016.11.22 07:30', '2016.11.22 08:00', false, undefined ],
"Week range. Working with Objects not Strings. from = new Date()" for "week 01-53/2 Mo-Su 07:30-08:00": FAILED, bad duration(s): 0,0, expected 1800000,0, bad intervals: 
(none)
expected:
[ '2016.11.22 07:30', '2016.11.22 08:00', false, undefined ],
0/2 tests passed. 2 did not pass.
```

i've modified the test to reset the timekeeper and turn it on again afterwards